### ISSUE TITLE
HDT - Capturable Overlay - broken in XSplit 

### ISSUE THREAD

**Issue by Cappster-Underscore** [HearthSim/Hearthstone-Deck-Tracker#2714]
After trying several times to re-setup HDT Overlay for my stream, I kept running into the same issue.

No matter which option I chose as the capture source, it always selected the Hearthstone Deck Tracker window, not the Overlay window.

![screen shot](https://cloud.githubusercontent.com/assets/20732313/17265152/8caf9aae-55ba-11e6-8bc2-44b3dbe49e86.jpg)

**Comment by Apreche** [comment]
I have the same problem.

**Comment by riQQ** [comment]
Have you checked this: https://github.com/HearthSim/Hearthstone-Deck-Tracker/wiki/Streaming-Instructions?

**Comment by Apreche** [comment]
@riQQ Yes, those are the exact instructions I followed. The problem isn't that it's not working. It's not working in a particular unfixable way.

**Comment by Bert-Proesmans** [comment]
Is this related to #3364, but then XSplit maybe automatically falls back to the main window because a lack of visual contents?
I'm gonna debug this during the week, but i won't make any promises.

**Comment by Bert-Proesmans** [comment]
So, i installed XSplit and found the same (or a similar) issue. I solved mine by enabling `Stick to title` for the overlay source. -> This was enabled by default in older versions but that got changed. 

The streaming instructions on the WIKI are updated.

I went a bit further and investigated logs, but it doesn't seem any input source related information get written to the log. If the above solution isn't a fix you should probably ask on their user forum.
